apps/backoffice-v2/src/pages/MerchantMonitoringCreateCheck/MerchantMonitoringCreateCheck.page.tsx

```diff
@@ -52,7 +52,6 @@ export const MerchantMonitoringCreateCheckPage: FunctionComponent = () => {
         </Link>
       </div>
       <h1 className="pb-5 text-2xl font-bold">Create Merchant Check</h1>
-      <h3 className="pb-5 text-sm">Registered Company Name</h3>
```

Comment:
Unrelated fix: this headline is not related to the page, it was mistakenly added.